IanGRap/Squish
# Controller Bugfixes

Contributors: Austin

Goals:
- Fix the bug where it is very difficult to dash straight up in the air
- Fix the bug where Amazon basics controllers sometimes won't stop vibrating

What changed:
- In player dash, I changed the way the dash direction was calculated to make it more exact.
- Changed the default driver for windows machines.
    - Alfredo and I tested this briefly the other day and had no issues but we couldn't test for too long.

How to test:
- Load up the lobby or the match and dash around.
- Is it easier for you to dash in the direction you intend?

- Play the game with an Amazon Basics controller (the white XBox One contollers in the game lab) for a little while and make sure you don't get the bug where the controller won't stop vibrating.
